dir.props

```diff
@@ -67,6 +67,7 @@
   <!-- list of nuget package sources passed to dnu -->
   <ItemGroup>
     <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
+
```

Comment:
I had put another package source there for testing on Linux and when I removed it I missed deleting everything.